core/src/main/scala/org/http4s/jdkhttpclient/JdkHttpClient.scala

```diff
@@ -58,35 +56,51 @@ object JdkHttpClient {
     *   cannot be set by the user. By default, the set of restricted headers of the OpenJDK 11 is
     *   used.
     */
+  @deprecated("Use `make` which does not return a `Resource`", "0.7.1")
   def apply[F[_]](
       jdkHttpClient: HttpClient,
       ignoredHeaders: Set[CIString] = restrictedHeaders
-  )(implicit F: Async[F]): Resource[F, Client[F]] = Dispatcher[F].map { dispatcher =>
```

Comment:
We no longer need a dispatcher for the client. FS2 now uses an internal "sequential" dispatcher that can be more efficient because it is scoped to the particular stream/publisher instead of shared globally.